apps/web/pages/api/twilio/webhook.ts

```diff
@@ -121,7 +121,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 
   if (!orgId) {
     orgId = await getOrgIdFromMemberOrTeamId({
-      memberId: parsedUserId,
+      ...(!parsedTeamId ? { memberId: parsedUserId } : {}),
```

Comment:
If we have the teamId, we only want to know if the team is part of an organization. 

There is a possibility that the parsedUserId is part of a platform org, but the provided teamId is not from this org. This can cause team credits to be assigned wrongly to the platform org

Reply:
Huh this is really tricky. I think this change makes sense, but if this pattern happens elsewhere more often, we could have another function that takes parameters like...

```
somethingSomething({
  type, // 'team' | 'user',
  id,    // number
})
```

then it will be exclusive and more readable. (Not suggesting we do this, but just sharing my chain of thought for future reference)